paasta_tools/mesos_tools.py

```diff
@@ -22,6 +22,7 @@
 from kazoo.client import KazooClient
 from mesos.cli import util
 from mesos.cli.exceptions import SlaveDoesNotExist
+from mesos.cli.master import CURRENT
```

Comment:
mesos_tools imports mesos.cli master at line 79 and then uses master.CURRENT.xyz afterwards. You may want to do the same.

 77 MESOS_MASTER_PORT = 5050
 78 MESOS_SLAVE_PORT = '5051'
 79 from mesos.cli import master  # noqa